### PR TITLE
Dev

### PR DESCRIPTION
**Pull request by dbourasseau**
- add load button  
- fix bug « no solution found »

### CODE REVIEW

File: gentelella/app/js/loadTopo.js [hunk 1]
```diff
@@ -98,6 +100,7 @@ function sendTopo() {
 
     function onLoad(e) {
         if (req.status >= 200 && req.status <= 299) {
```
Comment (nherbaut) [inline gentelella/app/js/loadTopo.js:102]: what is the real code returned by the resquest? why do we need to filter of the 200+ range?

File: simu-dockermgt/src/main/java/com/nh/db/ml/simuservice/dockermgt/service/imp/DockerServiceImp.java [hunk 2]
```diff
@@ -149,11 +149,11 @@ public void createSvgFromGrid(Grid grid) {
 				.withCmd("chmod", "737", "/opt/simuservice/offline/results/res.svg").exec();
 
 		dockerClient.startContainerCmd(container.getId()).exec();
-		dockerClient.waitContainerCmd(container.getId()).exec(new WaitContainerResultCallback());
+		dockerClient.waitContainerCmd(container.getId()).exec(new WaitContainerResultCallback()).awaitStatusCode();
```
Comment (nherbaut) [inline simu-dockermgt/src/main/java/com/nh/db/ml/simuservice/dockermgt/service/imp/DockerServiceImp.java:152]: duplicated code

File: gentelella/app/js/loadTopo.js (hunk 1 above)
Comment (dbourasseau) [reply gentelella/app/js/loadTopo.js:102]: 202 Accepted
No real reason of the range

File: simu-dockermgt/src/main/java/com/nh/db/ml/simuservice/dockermgt/service/imp/DockerServiceImp.java (hunk 2 above)
Comment (dbourasseau) [reply simu-dockermgt/src/main/java/com/nh/db/ml/simuservice/dockermgt/service/imp/DockerServiceImp.java:152]: added to the trello (need to by sorted)

**Pull request merged**